test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -287,14 +288,14 @@ describe('33acrossAnalyticsAdapter:', function () {
 
         performStandardAuction();
         performStandardAuction();
-        sandbox.clock.tick(this.defaultTimeout + 1);
+        sandbox.clock.tick(this.defaultTimeout + 1000);
 
         assert.calledTwice(navigator.sendBeacon);
       });
     });
 
     context('when an AUCTION_END event is received before BID_WON events', function () {
-      it('it sends a report with the bids that have won', function () {
+      it('it sends a report with the bids that have won after timeout', function () {
```

Comment:
I think it's better not to use `it` at the start of the title.. let's use the name of the function as part of the sentence.

` it('sends a report with the bids ...` 

please change other unit tests titles as well.

Reply:
Sure. I put it in so that it would read better when it prints, but I don't really have a strong preference about this.